script.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+// Create a car object, add a color property to it with the value equals 'black'
+const carObj = {};
+carObj.color = "black"
+console.log(carObj);
+
+// Change the color property of the car object to 'green'
+carObj.color = "green";
+console.log(carObj);
+
+// Add the power property to the car object, which is a function and displays the engine power to the console
+carObj.power = function(engPow){
+    console.log("red");
+};
+
+//Pears and apples are accepted to the warehouse, write a function that returns the result of adding the number of accepted pears and apples
+function sum (pearsNum, applesNum) {
+    let result = pearsNum + applesNum
+    return result;
+};
+console.log(sum(7,4));
+
+//Your name is saved in the payment terminal, write a function to define the name in the terminal (if you entered your name, then hello + name, if not, then there is no such name)
+function verify(){
+    let name = prompt ("Write your name");
+    if (name == "Jack") {
+        alert(`Hello ${name}`);
+    } else {
+        alert(`There is no such name: ${name}`);
+    };
+};
+
```

Comment:
do not use promt and alert functions, they are for browser environment
let's pass name into your function as parametr